mmdet/datasets/transforms/wrappers.py

```diff
@@ -16,12 +16,15 @@ class MultiBranch(BaseTransform):
     Generate multiple data-augmented versions of the same image.
 
     Args:
+        branch_field (list): List of branch names.
```

Comment:
might have more description of branch_pipelines because essentially the input is **branch_pipelines, it might mislead users to input `branch_pipelines=dict(xxx)`

Reply:
add some examples in the docstring to indicate usages